From 681462eca8bd87383c9cb8c0bc34ebad827c4421 Mon Sep 17 00:00:00 2001
From: Hannes Reinecke <hare@suse.de>
Date: Thu, 11 Dec 2014 15:46:26 +0100
Subject: [PATCH] 40network: add missing _arch variable declaration

The module_setup.sh script was missing an '_arch' declaration,
causing network not to be installed on s390.

Signed-off-by: Thomas Renninger <trenn@suse.de>
---
 modules.d/40network/module-setup.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/40network/module-setup.sh b/modules.d/40network/module-setup.sh
index 55bee4c..f7e0a87 100755
--- a/modules.d/40network/module-setup.sh
+++ b/modules.d/40network/module-setup.sh
@@ -17,6 +17,7 @@ depends() {
 # called by dracut
 installkernel() {
     # Include wired net drivers, excluding wireless
+    local _arch=$(uname -m)
 
     net_module_filter() {
         local _net_drivers='eth_type_trans|register_virtio_device|usbnet_open'
